scripts/clippy.args

```diff
@@ -13,6 +13,8 @@ allowed_lints=(
     "-A" "clippy::needless_lifetimes"
     # This ends up being platform dependent, e.g. the Instant type
     "-A" "clippy::trivially-copy-pass-by-ref"
+    # Allow Option<Option<...>>.
+    "-A" "clippy::option-option"
```

Comment:
Seems like bad sign when we have to override a clippy rule to allow this.  Isn't there a better alternative?